src/ThemedStyleSheet.js

```diff
@@ -32,6 +32,7 @@ function resolve(...styles) {
     && typeof performance !== 'undefined'
     && performance.mark !== undefined
   ) {
+    performance.clearMarks('react-with-styles.resolve.start');
```

Comment:
the above if checks (throughout) should be amended to also ensure that `performance.clearMarks` is a function